lib/oauth2.dart

```diff
@@ -50,7 +50,7 @@ const defaultApplicationId = "com.example.app";
 
 /// Get the custom uri scheme of the app for the current platform.
 String getPlatformCustomUriScheme({
-  @Deprecated("Use applicationId instead.") String androidApplicationId = defaultApplicationId,
+  @Deprecated("Use 'applicationId' instead.") String androidApplicationId = defaultApplicationId,
```

Comment:
You should not set the default value for `androidApplicationId` and `applicationId` and set them nullable to compare them with `null` instead of the default value. Use the default value only if none of them is defined: 
```dart
applicationId = applicationId ?? androidApplicationId ?? defaultApplicationId;
```